setup.py

```diff
@@ -12,14 +12,17 @@
 with open(path.join(HERE, PACKAGE_NAME, "base.py"), encoding="utf-8") as fp:
     VERSION = re.search('__version__ = "([^"]+)"', fp.read()).group(1)
 
+
+def read_requirements(filename):
+    return open(filename).read().strip().splitlines()
```

Comment:
i think this should be replaced with the context manager
```py
def read_requirements(filename):
     with open(filename, "r", encoding="utf-8") as req:
             return req.read().strip().split()
```

Reply:
Thanks, updated :)